src/v2/Apps/Artwork/Components/ArtworkArtistSeries/ArtistSeriesArtworkRail.tsx

```diff
@@ -29,14 +37,48 @@ export const ArtistSeriesArtworkRail: React.FC<Props> = ({
 
   const isMobile = getENV("IS_MOBILE") === true
 
+  const trackArtworkClick = (artworkSlug, artworkID, index) => {
```

Comment:
@eessex I ended up not using a helper here-- between these two events, the `ClickedArtworkGroup` type ensures that the `action` is correct, and everything else is specific to this component / context. I don't _think_ the `clickedEntityHeader` gives you too much here except verifying the `destinationPageOwnerType` (but I would need to modify that anyways-- I'm not sure it adds too much).

I could add a helper that's specific to this page/component, but that felt like overkill since it'll only be used in one place. 

I'm newish to cohesion, so let me know if I'm thinking about this the wrong way!

Reply:
Interesting! Probably there should be some sort of `clickedArtworkInShelf` or something helper, which admittedly might only include the `action` and `type: 'thumbnail'`. That doesn't need to be in this PR I don't think?

Reply:
The `type` is different between the two events here, so the only thing that's the same is `action` (which is enforced by the `ClickedArtworkGroup` type)

Reply:
Added a follow up PR here to split these out and force more required properties: https://github.com/artsy/cohesion/pull/96